core/src/Kokkos_BitManipulation.hpp

```diff
@@ -120,7 +347,7 @@ bit_cast(From const& from) noexcept {
 template <class T>
 KOKKOS_FUNCTION constexpr std::enable_if_t<std::is_integral_v<T>, T> byteswap(
```

Comment:
I suppose while we are at it I could use require clauses instead.
Let me know if you would like me to do that as well.

Reply:
Not required ;-) but also in this case you just could use the concept or? i.e. ```template<std::integral T>```